src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/AssemblyBuilderImpl.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection.Metadata;
+using System.Reflection.Metadata.Ecma335;
+using System.Reflection.PortableExecutable;
+
+namespace System.Reflection.Emit
+{
+    internal sealed class AssemblyBuilderImpl : AssemblyBuilder
+    {
+        private bool _previouslySaved;
+        private readonly AssemblyName _assemblyName;
+        internal readonly Assembly _coreAssembly;
```

Comment:
```suggestion
        private readonly Assembly _coreAssembly;
```
?
I do not see why it needs to be internal